### PR TITLE
chore(github): update to improve wording on feature template

### PR DESCRIPTION
**Pull request by awentzel**
Update the `.github/` template features.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ about: Suggest an idea for this project
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**What is your feature request? Please describe.**
+A clear and concise description of the requested feature and how this will improve the user experience. Ex. I'm always frustrated when [...]
```
Comment (nicholasrice) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: This is a little confusing to me - who is the user?

Maybe reword to something along the lines of: "A clear and concise description of the feature and the benefits it would have."

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ about: Suggest an idea for this project
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**What is your feature request? Please describe.**
+A clear and concise description of the requested feature and benefits it would have.
```
Comment (nalogiudice) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: nit: "...and the benefits it would provide."

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 3]
```diff
@@ -4,10 +4,10 @@ about: Suggest an idea for this project
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**What is your feature request? Please describe.**
+A clear and concise description of the requested feature and benefits it would provide.
 
-**Describe the solution you'd like**
+**Describe the solution you'd like**s
```
Comment (nalogiudice) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: typo 's'
